meta-resin-common/recipes-core/systemd/systemd/resin.target

```diff
@@ -0,0 +1,2 @@
+[Unit]
+Requires=resin.target
```

Comment:
This is present in my local repo, it was not pushed because my laptop has crashed over the weekend.